pkg/service/v0/settings.go

```diff
@@ -121,3 +133,136 @@ func generateBundleProfileRequest() settings.SaveBundleRequest {
 		},
 	}
 }
+
+func generateProfilePermissionsRequests() []settings.AddSettingToBundleRequest {
+	// TODO: we don't want to set up permissions for settings manually in the future. Instead each setting should come with
+	// a set of default permissions for the default roles (guest, user, admin).
+	return []settings.AddSettingToBundleRequest{
+		{
+			BundleId: ssvc.BundleUUIDRoleAdmin,
+			Setting: &settings.Setting{
+				Id:          "7d81f103-0488-4853-bce5-98dcce36d649",
+				Name:        "language-create",
+				DisplayName: "Permission to set the language",
+				Resource: &settings.Resource{
+					Type: settings.Resource_TYPE_SETTING,
+					Id:   settingUUIDProfileLanguage,
+				},
+				Value: &settings.Setting_PermissionValue{
+					PermissionValue: &settings.Permission{
+						Operation:  settings.Permission_OPERATION_CREATE,
```

Comment:
so setting the language is an act of creation ? :thinking:

Reply:
We have CREATE and UPDATE as separate operations. There are permissions for both. CREATE on first request, UPDATE on subsequent requests. The operation is one of the two things I wanted to discuss with Jörn, both Alex and myself were not too happy with it so far... but still, this reflects our current iteration. 🤷‍♂️

Reply:
https://github.com/owncloud/ocis-settings/issues/52